### PR TITLE
Modify the GetGlyph() interface to ensure that const2bit works properly

### PR DESCRIPTION
**Pull request by sago35**
Use `Glypher inteface` for cases where different rendering, such as const2bit, is required.
No heap allocations occur.
Additionally, this change makes const2bit available. (#43)

### CODE REVIEW

**Comment by sago35** [comment]
For example, you can create a font using const2bit as shown below.
Since it's a 2-bit font, you'll generally need a display with 4 or more colors.

```
$ go run ./cmd/tinyfontgen-ttf --size 16 --verbose ./_fonts/NotoSansJP-Regular.otf
```

https://fonts.google.com/noto/specimen/Noto+Sans+JP

File: tinyfont.go [hunk 1]
```diff
@@ -120,6 +120,6 @@ func LineWidth(f Fonter, str string) (innerWidth uint32, outboxWidth uint32) {
 }
 
 // GetGlyph returns the glyph corresponding to the specified rune in the font.
-func GetGlyph(f Fonter, r rune) Glyph {
+func GetGlyph(f Fonter, r rune) Glypher {
```
Comment (sago35) [inline tinyfont.go:123]: This will revert the changes made in 05eb156cff8b312112ba7e43e414d7072da89950. The only cases where this change could cause issues would be if this function has been used after 05eb156cff8b312112ba7e43e414d7072da89950, but I believe such instances are rare.

**Comment by sago35** [comment]
I believe it was originally ready for review, but after a self-check, I only made corrections to func GetGlyph. Everything else should be fine.

**Comment by deadprogram** [comment]
OK got it. Thanks for the improvement @sago35 now merging.

Note that I switched this PR to `dev` branch.
